--- apache-mod_rails-3.0.5/build/config.rb~	2011-03-11 12:11:27.000000000 +0100
+++ apache-mod_rails-3.0.5/build/config.rb	2011-03-30 22:02:43.754874026 +0200
@@ -24,9 +24,10 @@
 OPTIMIZE = boolean_option("OPTIMIZE")
 CC       = string_option("CC", "gcc")
 CXX      = string_option("CXX", "g++")
+CXXFLAGS = string_option("CXXFLAGS", "")
 # TODO: consider -fcommon
 if OPTIMIZE
-	OPTIMIZATION_FLAGS = "#{PlatformInfo.debugging_cflags} -O2 -DBOOST_DISABLE_ASSERTS".strip
+	OPTIMIZATION_FLAGS = "#{PlatformInfo.debugging_cflags} -DBOOST_DISABLE_ASSERTS".strip
 else
 	OPTIMIZATION_FLAGS = "#{PlatformInfo.debugging_cflags} -DPASSENGER_DEBUG -DBOOST_DISABLE_ASSERTS".strip
 end
@@ -32,10 +33,11 @@
 
 # Extra compiler flags that should always be passed to the C/C++ compiler.
 # Should be included last in the command string, even after PlatformInfo.portability_cflags.
-EXTRA_CXXFLAGS = "-Wextra -Wno-unused-parameter -Wno-parentheses -Wpointer-arith -Wwrite-strings -Wno-long-long"
+EXTRA_CXXFLAGS = "-Wextra -Wno-unused-parameter -Wno-parentheses -Wpointer-arith -Wwrite-strings -Wno-long-long"
 EXTRA_CXXFLAGS << " -Wno-missing-field-initializers" if PlatformInfo.compiler_supports_wno_missing_field_initializers_flag?
 EXTRA_CXXFLAGS << " -mno-tls-direct-seg-refs" if PlatformInfo.requires_no_tls_direct_seg_refs? && PlatformInfo.compiler_supports_no_tls_direct_seg_refs_option?
 EXTRA_CXXFLAGS << " #{OPTIMIZATION_FLAGS}" if !OPTIMIZATION_FLAGS.empty?
+EXTRA_CXXFLAGS << " #{CXXFLAGS}" if !CXXFLAGS.empty?
 
 # Extra linker flags that should always be passed to the linker.
 # Should be included last in the command string, even after PlatformInfo.portability_ldflags.
--- apache-mod_rails-3.0.5/ext/libev/configure.ac~	2011-03-11 12:11:27.000000000 +0100
+++ apache-mod_rails-3.0.5/ext/libev/configure.ac	2011-03-31 11:39:27.773936137 +0200
@@ -8,10 +8,6 @@
 AC_PROG_INSTALL
 AC_PROG_LIBTOOL
 
-if test "x$GCC" = xyes ; then
-  CFLAGS="$CFLAGS -O3"
-fi
-
 m4_include([libev.m4])
 
 AC_CONFIG_FILES([Makefile])
