--- passenger-release-4.0.50/ext/common/ResourceLocator.h~	2014-08-26 16:15:57.000000000 +0200
+++ passenger-release-4.0.50/ext/common/ResourceLocator.h	2014-09-04 21:47:26.617938160 +0200
@@ -78,12 +78,12 @@
 		} else {
 			string root = rootOrFile;
 			binDir              = root + "/bin";
-			agentsDir           = root + "/buildout/agents";
-			helperScriptsDir    = root + "/helper-scripts";
-			resourcesDir        = root + "/resources";
-			docDir              = root + "/doc";
-			rubyLibDir          = root + "/lib";
-			nodeLibDir          = root + "/node_lib";
+			agentsDir           = root + "/@@LIB@@/phusion-passenger/agents";
+			helperScriptsDir    = root + "/share/phusion-passenger/helper-scripts";
+			resourcesDir        = root + "/share/phusion-passenger/resources";
+			docDir              = root + "/share/doc";
+			rubyLibDir          = root + "@@RUBYLIBDIR@@";
+			nodeLibDir          = root + "/share/phusion-passenger/node_lib";
 		}
 	}
 
