--- passenger-release-3.0.19/lib/phusion_passenger/platform_info/apache.rb~	2012-12-20 14:23:00.000000000 +0200
+++ passenger-release-3.0.19/lib/phusion_passenger/platform_info/apache.rb	2013-05-01 22:05:55.625131364 +0300
@@ -58,6 +58,9 @@
 	# The absolute path to the 'apachectl' or 'apache2ctl' binary, or nil if
 	# not found.
 	def self.apache2ctl
+		if env_defined?("APACHECTL")
+			return ENV["APACHECTL"]
+		end
 		return find_apache2_executable('apache2ctl', 'apachectl2', 'apachectl')
 	end
 	memoize :apache2ctl
@@ -83,6 +86,9 @@
 
 	# The Apache version, or nil if Apache is not found.
 	def self.httpd_version
+		if env_defined?("HTTPD_VERSION")
+			return ENV["HTTPD_VERSION"]
+		end
 		if httpd
 			`#{httpd} -v` =~ %r{Apache/([\d\.]+)}
 			return $1
